account_bank_statement_import_txt_xlsx/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Account Bank Statement Import TXT XLSX",
     "summary": "Import TXT/CSV or XLSX files as Bank Statements in Odoo",
-    "version": "12.0.2.0.2",
+    "version": "12.0.2.0.3",
```

Comment:
Normally you don't change the version manually, as the bot does it automatically when merging.

Reply:
Oh, sorry!  
My bad... 😥